src/streams/TopicItem.js

```diff
@@ -45,20 +54,25 @@ export default function TopicItem(props: Props) {
   const backgroundData = useSelector(state => ({
     auth: getAuth(state),
     mute: getMute(state),
+    streams: getStreamsById(state),
     subscriptions: getSubscriptions(state),
+    unreadStreams: getUnreadStreams(state),
     ownUser: getOwnUser(state),
     flags: getFlags(state),
   }));
 
+  const currentStream = [...backgroundData.streams.entries()].find(x => x[1].name === streamName);
```

Comment:
I wanted to de-structure this similar to
```javascript
const [currentStreamId, currentStream] = [...backgroundData.streams.entries()].find(x => x[1].name === streamName);    
```
Because it seemed more clean, but flow doesn't allow this because it thinks that this can potentially be `undefined`. Is their a way to handle that?